src/arguments.js

```diff
@@ -181,8 +208,9 @@ const checkFlagsByExamples = (section, context) => {
  * @return {string} - result string
  */
 const removeExtraArgumentNames = (string, argument) => {
-    return removeChar(string,
-        [',', argument.longName, argument.shortName]).trim();
+    let remove = [','].concat(argument.longNames).concat(argument.shortNames);
```

Comment:
Mb better use `[',', argument.longNames, .. ]` ?

Reply:
@svoboda-rabstvo no, in that case array will be included in other array. 

For example:
``` 
longNames = ['--color', '--no-color'];
shortNames = ['-c', '-n'];
```
remove array will be:
```
remove = [',', ['--color', '--no-color'], ['-c', '-n']];
```